### DISCUSSION TITLE
Can we add a new metrics related to  '$upstream_connect_time' for prometheus-nginxlog-exporter.

### DISCUSSION THREAD

**Discussion by Ais18** [martin-helmich/prometheus-nginxlog-exporter#249] [Q&A]
In 'log_format' we use '$upstream_connect_time', but this is not been used by any of the predefined metrics in prometheus-nginxlog-exporter. So to monitor the connect time is it possible to add a metrics for '$upstream_connect_time' .

Thanks

**Comment by martin-helmich** [comment]
Thanks for the (very reasonable) suggestion! I've created https://github.com/martin-helmich/prometheus-nginxlog-exporter/issues/252 to track this feature request and see if I'll get around to implementing this some time soon (in the mean time, PRs are welcome 🙂).
